raiden/network/transport/matrix/utils.py

```diff
@@ -185,6 +185,20 @@ def populate_userids_for_address(self, address: Address, force: bool = False):
                 ),
             )
 
+    def get_online_user_ids_for_address(self, address: Address) -> Set[str]:
+        online_user_ids = {
+            user_id
+            for user_id in self.get_userids_for_address(address)
+            if self.get_userid_presence(user_id) == UserPresence.ONLINE or UserPresence.UNKNOWN
```

Comment:
The condition isn't right. It will always be True since the `or` clause only evaluates `UserPresence.UNKNOWN` (enum items are always true).